app/views/pages/articles/2016-11-17-ruby-au-elections.html.md

```diff
@@ -0,0 +1,48 @@
+# Ruby Au Elections
+
+
+We have 5 positions open, President and Secretary, as well as two full-term and one half-term general position. Below is a brief description of each position, and a link to a nomination form to help streamline the meeting. If you are nominating someone other than yourself, please be sure to check with them first.
+
+Each committee member is expected to attend a monthly 1 hour online meeting (scheduled around all members other commitments), and to have regular access to Slack, 1password and Google docs. These are minimal requirements for staying in contact with the rest of the committee and accessing required files and login information. The time outputs outlined below are loose, with more time often needed at crucial points throughout the year and less at other times. Critical points are usually: turnover of committee members (after each Rails Camp), the weeks before RubyConf and Rails Camps and the turnover of the financial year.
+
+
+[nominations](https://docs.google.com/forms/d/e/1FAIpQLSdK7a3N3-5XT5LRqZIVBqEW1NgcxNTZJqaVs7kHwgwyZ1wDog/viewform)
+
+
+## President
+The president guides the overall direction of the organisation. They make the final call on major decisions and delegate work to general members. They are in charge of making sure the committee is running smoothly and committee members are meeting their obligations. Occasionally they need to mediate interpersonal issues within the committee or address code of conduct breaches within the member base.
+
+The president is in charge of chairing the general meetings, though this is often delegated to a trusted member of the organisation. They also cast the deciding vote in the instance of a draw.
+
+Time output: 5+ hours a week
+
+
+## Secretary
+The secretary is responsible to ensure that formal legal and governance requirements of running an association are maintained. They also handle the nitty-gritty of day to day committee operation, such as keeping members in the loop on meetings, taking and distributing minutes and maintaining our online accounts. We also expect that they will be in charge of maintaining a new member database in the coming year.
+
+The secretary job is a big one, which can involve delegating work out to general members.
+
+Time output: 4-5 hours a week
+
+
+## General Members
+General members are tasked with supporting the current needs of the organisation in various ways. They might be on the organising team of an upcoming camp or conference, heading a subcommittee to work on a particular aspect of the organisation like the website/sponsorship or handling the social media and other member communication. We try to utilise the strengths and interests of each member to their best effect, and to allow them a fair amount of autonomy in their area of focus.
+
+Time output: 2-3 hours a week
+
+
+The main goals of the organisation for 2017 are:
+Streamline onboarding of new committee members
```

Comment:
Convert these into dot-points so that they render clearer.

![image](https://cloud.githubusercontent.com/assets/2687/20382317/7b70b02e-acff-11e6-9af4-cbcb1d4dded4.png)